### ISSUE TITLE
Bug in evalution of condition strings

### ISSUE THREAD

**Issue by ludgerh** [ludgerh/cam-ai#19]
```
cam_ai@CamAIRaspiAc:~ $ tail -f ~/cam-ai/data/logs/c_server.err
Traceback (most recent call last):
  File "/home/cam_ai/cam-ai/eventers/c_eventers.py", line 241, in display_events
    if resolve_rules(self.cond_dict[self.last_cond_ed], predictions):
  File "/home/cam_ai/cam-ai/eventers/c_event.py", line 94, in resolve_rules
    return(eval(cond_str))
  File "<string>", line 1
    (False
         ^
SyntaxError: unexpected EOF while parsing
```

**Comment by ludgerh** [comment]
Reproducable: Create one single condition and hit the right array. Then save...

**Comment by ludgerh** [comment]
Fixed with 0.8.9
